sample-apps/react-native/dogfood/reactNativePermissionsIOS.json

```diff
@@ -0,0 +1,6 @@
+[
+  "BluetoothPeripheral",
+  "Camera",
+  "Microphone",
+  "Notifications"
+]
```

Comment:
bluetooth and notifications not necessary?

Reply:
Actually you're right